channels/es.m3u

```diff
@@ -345,8 +349,8 @@ https://mdslivehlsb-i.akamaihd.net/hls/live/875601/niusdiario/03.m3u8
 https://mdslivehlsb-i.akamaihd.net/hls/live/875601-b/niusdiario/01.m3u8
 #EXTINF:-1 tvg-id="NoroesteTV.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Noroeste TV (720p) [Not 24/7]
 https://cloudvideo.servers10.com:8081/8032/index.m3u8
-#EXTINF:-1 tvg-id="NORTEvision.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",NORTEvisión [Offline]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://facebook.com/1918671788399481/videos/3060444244223429/
+#EXTINF:-1 tvg-id="NORTEvision.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",NORTEvisión
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://facebook.com/1918671788399481/videos/3060444244223429/
```

Comment:
Not quite sure this is the correct FB stream. It leads me to a 18 minute-long video.

Reply:
Fixed.